docs/migration.md

```diff
@@ -0,0 +1,11 @@
+This file contains SQL scripts for migrating data from the Athene-created version of Ilmomasiina.
```

Comment:
I think we should also have some kind of automatic migration system too, as we will anyways run into the need of doing SQL migrations in the development process.

Reply:
Of course providing one-time migration script for people switching to this version is alright, but could it also be done using some tool that will also provide up/down migrations for future versions?

Reply:
Yes and no. Sequelize has its own [migration system](https://sequelize.org/master/manual/migrations.html) but it doesn't support TypeScript and, as far as I can tell, relies somewhat on their CLI-generated models.

I'll investigate up/down migrations when we get closer to deploying Ilmomasiina, but I don't think it's worth doing at this point when changes are happening rapidly.